it/controllers/gains/PolicySummaryControllerISpec.scala

```diff
@@ -143,10 +143,10 @@ class PolicySummaryControllerISpec extends IntegrationTest {
   }
 
   ".submit" should {
-    "redirect to the gains summary page" in {
+    "redirect to the gains summary page when it has prior data and same policy reference number" in {
       lazy val result: WSResponse = {
         clearSession()
-        populateWithSessionDataModel(Seq(completePolicyCyaModel))
```

Comment:
If you copy this test and change `populateWithSessionDataModel(Seq(completePolicyCyaModel.copy(policyNumber = Some("abc123"))))`
to
`populateWithSessionDataModel(Seq())`
you can also test the getOrElse on line 98 of the controller.